solidity/dashboard/src/pages/liquidity/LiquidityPage.jsx

```diff
@@ -16,16 +16,82 @@ import {
 import Banner from "../../components/Banner"
 import { useHideComponent } from "../../hooks/useHideComponent"
 import { gt } from "../../utils/arithmetics.utils"
-
 import KeepOnlyPool from "../../components/KeepOnlyPool"
 
+const cards = [
+  {
+    id: "TBTCV2_SADDLE",
+    title: LIQUIDITY_REWARD_PAIRS.TBTCV2_SADDLE.label,
+    liquidityPairContractName:
+      LIQUIDITY_REWARD_PAIRS.TBTCV2_SADDLE.contractName,
+    MainIcon: Icons.TBTC,
+    SecondaryIcon: Icons.Saddle,
+    viewPoolLink: LIQUIDITY_REWARD_PAIRS.TBTCV2_SADDLE.viewPoolLink,
+    pool: LIQUIDITY_REWARD_PAIRS.TBTCV2_SADDLE.pool,
+    lpTokens: LIQUIDITY_REWARD_PAIRS.TBTCV2_SADDLE.lpTokens,
+    wrapperClassName: "tbtc-v2-saddle",
```

Comment:
I think we should style the logos in this card and fill the tbtc logo with white color (we did the same in TBTC-SADDLE pool):

```
.tbtc-saddle& {
    .main-icon {
      circle {
        fill: @white;
      }
    }
  }
```

in `liquigidy-page.less`.

Reply:
c64211f